src/js/components/Footer/stories/DoubleFooter.js

```diff
@@ -37,12 +37,18 @@ export const DoubleFooter = () => (
       </Text>
       <Box flex />
     </Main>
-    <Footer background="dark-1" pad="large">
+    <Footer
+      background="dark-1"
+      pad="large"
+      role="region"
```

Comment:
The accessibility panel is flagging `role="region"` as an issue
"ARIA role region is not allowed for given element"